src/components/studies/scheduler/IntroInfo.tsx

```diff
@@ -22,12 +23,12 @@ const useStyles = makeStyles((theme: Theme) =>
       fontSize: '18px',
       fontStyle: 'normal',
```

Comment:
we only need to specify fontStyle when it's not 'normal'. I don't think we should use % on the margins/padding/etc
and even in width for most cases